src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -29,5 +38,32 @@ const Footer = BaseFooter.extend({
 
 export default BaseView.extend({
   Body,
-  Footer
+  Footer,
+  createModelClass (currentViewState, optionUiSchemaConfig, settings) {
+    let ModelClass = BaseView.prototype.createModelClass.apply(this, arguments);
+
+    settings.parseSchema(currentViewState.uiSchema,
+      (schema) => {
+        currentViewState.uiSchema = schema;
```

Comment:
nit:
when the new schema is identical to old new (`schema === currentViewState.uiSchema`), no need to call super again.

Reply:
This is going to take more than a triple equal check.
Will need to deep clone the uiSchema and then deep comparison.